bitnami/postgresql/templates/svc-headless.yaml

```diff
@@ -27,5 +26,4 @@ spec:
     - name: tcp-postgresql
```

Comment:
I need to better understand the use case of uniqueServices vs headless services

Reply:
Honestly.. Me neither.. It seems that the chart is currently reusing the same headless svc to create unique FQDN(s) for both the "primary" and "read-only" replicas, but I guess we could create a different headless svc per replica type and remove the "uniqueServices" thing.